modules/nf-core/merfin/hist/tests/main.nf.test.snap

```diff
@@ -0,0 +1,184 @@
+{
+    "Multiple samples - sarscov2 + bacteroides_fragilis - paired-end illumina reads": {
+        "content": [
+            [
+                [
+                    {
+                        "id": "test_fragilis",
+                        "single_end": false
+                    },
+                    "test_fragilis.hist:md5,3380e6f8b82739edc44b45ac6b50525e"
+                ],
+                [
+                    {
+                        "id": "test_sars",
+                        "single_end": false
+                    },
+                    "test_sars.hist:md5,564c5dc22b9a10041b248405bb2b15d0"
+                ]
+            ],
+            [
+                "versions.yml:md5,078b89694148fc55475e89987173ac4e",
+                "versions.yml:md5,078b89694148fc55475e89987173ac4e"
```

Comment:
This is weird. This shouldn't be duplicated. Can you investigate this please?

Reply:
I took a look and the duplications occur for the tests that use multiple samples. Is it not because of that?

Reply:
Ah, understood.